### PR TITLE
Add lquerel as otel-arrow codeowner

### PR DESCRIPTION
**Pull request by jmacd**
**Description:** @lquerel is an owner of the OTel-Arrow repository so it makes sense to add him as CODEOWNER here for the OTel-Arrow components. 

**Link to tracking Issue:** #26491

### CODE REVIEW

**Review by codeboten — APPROVED** [review]
🎉

**Review by songy23 — COMMENTED** [review]
Please update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/otelarrowexporter/metadata.yaml#L10 as well

**Comment by songy23** [comment]
and https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/otelarrowreceiver/metadata.yaml#L10

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -254,7 +254,7 @@ receiver/nsxtreceiver/                                              @open-teleme
 receiver/opencensusreceiver/                                        @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
 receiver/oracledbreceiver/                                          @open-telemetry/collector-contrib-approvers @dmitryax @crobert-1 @atoulme
 receiver/osqueryreceiver/                                           @open-telemetry/collector-contrib-approvers @codeboten @nslaughter @smithclay
-receiver/otelarrowreceiver/                                         @open-telemetry/collector-contrib-approvers @jmacd @moh-osman3
+receiver/otelarrowreceiver/                                         @open-telemetry/collector-contrib-approvers @jmacd @moh-osman3 @lquerel
```
Comment (codeboten) [inline .github/CODEOWNERS:257]:
```suggestion
receiver/otelarrowreceiver/                                         @open-telemetry/collector-contrib-approvers @jmacd @moh-osman3
```

**Pull request merged**